Yardforce 500b

Pepeuch opened the pull request:
ajout des mpu 6500/9250/9255
optimisation du code par le remplacement de la fonction pow
quelques ajustements/réorganisations mineure dans le code.

Pepeuch committed:
charge value

Pepeuch committed:
ajout imu 6500/9250/9255
remplacement de la fonction pow par un equivalent moins gourmand en ressource cpu
deplacements des fonction de bases
	modified:   stm32/ros_usbnode/.vscode/settings.json
	modified:   stm32/ros_usbnode/include/imu/imu.h
	modified:   stm32/ros_usbnode/include/main.h
	modified:   stm32/ros_usbnode/src/imu/imu.c
	modified:   stm32/ros_usbnode/src/imu/mpu6050.c

Pepeuch committed:
modified:   stm32/ros_usbnode/include/imu/imu.h

Pepeuch committed:
optimise lsm6 and wt901

Pepeuch committed:
Update platformio.ini

Pepeuch committed:
modified:   stm32/ros_usbnode/include/board.h.template
	modified:   stm32/ros_usbnode/src/imu/lsm6.c
	modified:   stm32/ros_usbnode/src/imu/mpu6050.c
	modified:   stm32/ros_usbnode/src/imu/wt901.c

Pepeuch committed:
modified:   stm32/ros_usbnode/include/imu/imu.h
	modified:   stm32/ros_usbnode/src/imu/lsm6.c
	modified:   stm32/ros_usbnode/src/imu/mpu6050.c
	modified:   stm32/ros_usbnode/src/imu/wt901.c

Pepeuch committed:
modified:   stm32/ros_usbnode/include/imu/imu.h
	modified:   stm32/ros_usbnode/src/imu/lsm6.c
	modified:   stm32/ros_usbnode/src/imu/mpu6050.c
	modified:   stm32/ros_usbnode/src/imu/wt901.c

Pepeuch committed:
modified:   stm32/ros_usbnode/.vscode/settings.json
	modified:   stm32/ros_usbnode/src/imu/wt901.c

Pepeuch committed:
modified:   stm32/ros_usbnode/include/board.h
	modified:   stm32/ros_usbnode/include/board.h.template

quelqundev commented:
@Pepeuch n'est-ce pas un peu trop le big bang cette PR ? Peut-être devrais tu remettre à plus tard le refactoring car ca n'aide pas à la compréhension de ce qui change... :'(

Pepeuch committed:
Update board.h

cutoff correction

Pepeuch committed:
modified:   stm32/ros_usbnode/.vscode/settings.json
	modified:   stm32/ros_usbnode/include/board.h
    minor improvements
	modified:   stm32/ros_usbnode/src/adc.c
    float improvements
	modified:   stm32/ros_usbnode/src/charger.c
    float improvements
	modified:   stm32/ros_usbnode/src/i2c.c
    float improvements
	modified:   stm32/ros_usbnode/src/imu/imu.c
    float improvements
	modified:   stm32/ros_usbnode/src/imu/mpu6050.c

Pepeuch committed:
update template board for unifying onboard flashing
	modified:   stm32/ros_usbnode/include/board.h.template
